### PR TITLE
Adddogspicture

### PR DESCRIPTION
**Pull request by jocelynbaduria**

### CODE REVIEW

File: event.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
   "query": { "text": "serverless iopipe performance metrics workshop lambda amazon" }
 }
+{
```
Comment (shannonphu) [inline event.json]: remove duplicate?

**Review by shannonphu — CHANGES_REQUESTED** [review]
Can you fix the merge conflicts?

**Review by jocelynbaduria — COMMENTED** [review]
Delete duplicates

File: event.json (hunk 1 above)
Comment (jocelynbaduria) [reply event.json]: I remove the duplicate

**Pull request merged**